### PR TITLE
added Inkscape & IrfanView

### PR DESCRIPTION
**Pull request by EldoBam**
Hi,

I've added Inkscape and IrfanView.

version is get by webrequest and regex match.
URIs are generated dynamically by architectures and types. 

both are tested.

```
PS C:\Users\EldoBam> gea irfanview

Version Architecture Type URI                                                              
------- ------------ ---- ---                                                              
4.60    x86          exe  https://www.fosshub.com/IrfanView.html?dwl=iview460x86_setup.exe 
4.60    x64          exe  https://www.fosshub.com/IrfanView.html?dwl=iview460_x64_setup.exe

PS C:\Users\EldoBam> gea inkscape

Version Architecture Type         URI                                                                        
------- ------------ ----         ---                                                                        
1.2.1   x86          exe          https://inkscape.org/release/inkscape-1.2.1/windows/32-bit/exe/dl/         
1.2.1   x86          portable-app https://inkscape.org/release/inkscape-1.2.1/windows/32-bit/portable-app/dl/
1.2.1   x86          msi          https://inkscape.org/release/inkscape-1.2.1/windows/32-bit/msi/dl/         
1.2.1   x86          7z           https://inkscape.org/release/inkscape-1.2.1/windows/32-bit/7z/dl/          
1.2.1   x64          exe          https://inkscape.org/release/inkscape-1.2.1/windows/64-bit/exe/dl/         
1.2.1   x64          msi          https://inkscape.org/release/inkscape-1.2.1/windows/64-bit/msi/dl/         
1.2.1   x64          7z           https://inkscape.org/release/inkscape-1.2.1/windows/64-bit/7z/dl/ 
```

best regards

EldoBam

### CODE REVIEW

File: Evergreen/Manifests/Inkscape.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿{
+    "Name": "Inkscape",
+    "Source": "https://inkscape.org/",
+    "Get": {
+        "Uri": "https://inkscape.org/release",
```
Comment (aaronparker) [inline Evergreen/Manifests/Inkscape.json:5]: The approach with Evergreen is to not scrape HTML. When this page changes, there's a good chance the function will break

File: Evergreen/Manifests/IrfanView.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿{
+    "Name": "IrfanView",
+    "Source": "https://www.irfanview.com/",
+    "Get": {
+        "Uri": "https://www.irfanview.com/",
+        "MatchVersion": "<strong>version (\\d+(\\.\\d+){1,4})</strong>"
```
Comment (aaronparker) [inline Evergreen/Manifests/IrfanView.json:6]: The approach with Evergreen is to not scrape HTML. When this page changes, there's a good chance the function will break

File: Evergreen/Manifests/IrfanView.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿{
+    "Name": "IrfanView",
+    "Source": "https://www.irfanview.com/",
+    "Get": {
+        "Uri": "https://www.irfanview.com/",
```
Comment (aaronparker) [inline Evergreen/Manifests/IrfanView.json:5]: I need to update docs on the manifest structure, but this should be structured like the example here: https://github.com/aaronparker/evergreen/blob/2a54df73a5cc25aa0b65064dfed3d19b38a887fe/template/Template.json

File: Evergreen/Manifests/Inkscape.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+﻿{
+    "Name": "Inkscape",
+    "Source": "https://inkscape.org/",
+    "Get": {
```
Comment (aaronparker) [inline Evergreen/Manifests/Inkscape.json:4]: I need to update docs on the manifest structure, but this should be structured like the example here: https://github.com/aaronparker/evergreen/blob/2a54df73a5cc25aa0b65064dfed3d19b38a887fe/template/Template.json

**Comment by aaronparker** [comment]
Unfortunately I can't approve this PR as it is scraping HTML. The preferred approach is to see if these apps include an in-built updater and determine how that is finding the application version and download URLs.

**Comment by aaronparker** [comment]
I've checked both of these apps - neither has an in-built update check that can be used, and the download sites used for both apps don't provide any queries that could be replicated either. Unless there's some other sources, unfortunately these apps aren't good candidates for Evergreen.

**Comment by EldoBam** [comment]
Hi @aaronparker,
I will work on this. I just added an app for own purpose and tried to get along with looking how adobe and the other apps work.
thank you for your feedback.

EldoBam

**Comment by aaronparker** [comment]
Thanks for contributing. I appreciate it.

Check out how these examples work: https://github.com/aaronparker/evergreen#how-evergreen-works

**Pull request closed without merging**